CHANGELOG.md

```diff
@@ -584,3 +584,13 @@ Released with 1.0.0-beta.37 code base.
 ### Security
 -  Updated `got` lib version and fixed other libs using npm audit fix (#5178) (#5254)
 
+## [Unreleased]
```

Comment:
As the previous version (`1.7.5`) was not released yet, is it possible to add those changes to `1.7.5-rc.0`? Or would that violate the process?

Reply:
It was code freeze from our perspective, so we can't make any changes to it. The current workflow is to include the PR's in future release during the code freeze duration.